test/rewardsTest.js

```diff
@@ -318,5 +318,70 @@ describe("Rewards", () => {
         "Operator still ineligible",
       )
     })
+
+    it("returns correct amount of available rewards", async () => {
+      await rewards.addOperator(alice, 10)
+      await rewards.addOperator(bob, 10)
+      await rewards.addOperator(carol, 10)
+
+      // Reward to all
+      // Alice: 10; Bob: 10; Carol: 10
+      await rewards.payReward(30)
+
+      let aliceReward = await rewards.getAvailableRewards(alice)
+      expect(aliceReward).to.equal(10)
+      let bobRewards = await rewards.getAvailableRewards(bob)
+      expect(bobRewards).to.equal(10)
+      let carolRewards = await rewards.getAvailableRewards(carol)
+      expect(carolRewards).to.equal(10)
+
+      // Alice withdraws her rewards
+      // Alice: 0; Bob: 10; Carol: 10
+      await rewards.withdrawRewards(alice)
+
+      aliceReward = await rewards.getAvailableRewards(alice)
+      expect(aliceReward).to.equal(0)
+      bobRewards = await rewards.getAvailableRewards(bob)
+      expect(bobRewards).to.equal(10)
+      carolRewards = await rewards.getAvailableRewards(carol)
+      expect(carolRewards).to.equal(10)
+
+      // Reward to all
+      // Alice: 10; Bob: 20; Carol: 20
+      await rewards.payReward(30)
+
+      aliceReward = await rewards.getAvailableRewards(alice)
+      expect(aliceReward).to.equal(10)
+      bobRewards = await rewards.getAvailableRewards(bob)
+      expect(bobRewards).to.equal(20)
+      carolRewards = await rewards.getAvailableRewards(carol)
+      expect(carolRewards).to.equal(20)
+
+      // Bob goes ineligible.
+      await rewards.makeIneligible(bob, 10)
+
+      // Reward to all
+      // Alice: 20; Bob: 20; Carol: 30
+      await rewards.payReward(30)
```

Comment:
where do the other 10 go?

Reply:
They are put aside for ineligible operators:
https://github.com/keep-network/sortition-pools/blob/aecb3a5e395cd4a965e02a51e16481740b2e582a/contracts/Rewards.sol#L57-L59

and can be later withdrawn by the owner of `SortitionPool`:
https://github.com/keep-network/sortition-pools/blob/aecb3a5e395cd4a965e02a51e16481740b2e582a/contracts/Rewards.sol#L142-L147

In other words, it means that the DAO can "restore" and "recycle" them for future reward intervals.